src/picosvg/svg_types.py

```diff
@@ -505,6 +505,32 @@ def callback(subpath_start, curr_xy, cmd, args, prev_xy, prev_cmd, prev_args)
             self._add_cmd(cmd, *args)
         return self
 
+    def subpaths(self) -> Tuple[str, ...]:
+        subpaths = [SVGPath()]
+
+        def subpaths_callback(subpath_start, curr_pos, cmd, args, *_unused):
+            if cmd.upper() == "M":
```

Comment:
what if M is preceded by a Z (which I think is quite common)? You don't want to append two empty SVGPaths, one for Z and then another for M. Or maybe you are taking care of removing the empty ones after the callback when you return `tuple(s.d for s in subpaths if s.d)`?

Reply:
The filter at the end is supposed to dump any completely empty paths